python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml

```diff
@@ -29,6 +29,8 @@ keys:
               A rate of 16384 corresponds to an average sample of one per 16384 packets.
       destinations:
         type: list
+        required: true
```

Comment:
May be this is not a required key.

Reply:
Discussed with Claus and adding required true for now